### ISSUE TITLE
API for retrieving perifocal coordinates

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#108]
Right now the only way to get the position and velocity vectors expressed in the perifocal frame is through the untested `twobody.classical.rv_pqw` function, used only by `OrbitPlotter.plot`.

There should be an easier way to retrieve this coordinates. We could probably define our own coordinate frame, connected to GCRS, using `astropy.coordinates` as explained in http://docs.astropy.org/en/v1.1.1/coordinates/frames.html. Or simply provide low level conversion functions, as it was done in #215.

**Comment by Juanlu001** [comment]
Blocked on #109.

**Comment by Juanlu001** [comment]
Maybe we can add `inertial_to_pqw(r, v, source_body)` and `pqw_to_inertial(r, v, target_body)` to `coordinates.py` when #215 is finished? What do you think @anhiga?

Also, if we have low level functions, I think this is no longer blocked on #109.

**Comment by anhiga** [comment]
They are transformations, so the belong definitively in `coordinates.py`. Although, don't `pqw` coordinates need orbital elements to be defined?

**Comment by Juanlu001** [comment]
You can compute them in a simple way using orbital elements, but doesn't
mean it needs them. You can always do the conversion using `rv2coe`.

**Comment by Juanlu001** [comment]
Are we in time of doing this for 0.7 @anhiga or do you prefer to postpone it to 0.8?

**Comment by anhiga** [comment]
This is the current state: https://github.com/poliastro/poliastro/pull/228.

**Comment by anhiga** [comment]
I think we should postpone it to 0.8, since it is related to #218.
We could maybe add the transformation, but without using it in plotting.

**Comment by Juanlu001** [comment]
Sure, the description of the issue is a bit vague but in reality it only concerns the conversion. The simplification of plotting should be discussed only in #218, I'm updating this one accordingly.

**Comment by Juanlu001** [comment]
Closed by #228.
